src/bundle/Templating/Twig/Extension/RichTextConfigurationExtension.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformRichTextBundle\Templating\Twig\Extension;
+
+use EzSystems\EzPlatformRichText\API\Configuration;
+use Twig\Extension\AbstractExtension;
+use Twig\Extension\GlobalsInterface;
+
+/**
+ * Twig extension exposing RichText Configuration as ez_richtext_config global Twig variable.
+ *
+ * @internal To access configuration use \EzSystems\EzPlatformRichText\API\Configuration\ProviderService
+ */
+final class RichTextConfigurationExtension extends AbstractExtension implements GlobalsInterface
+{
+    /** @var \EzSystems\EzPlatformRichText\API\Configuration\ProviderService */
+    private $configurationProvider;
+
+    public function __construct(Configuration\ProviderService $configurationProvider)
+    {
+        $this->configurationProvider = $configurationProvider;
+    }
+
+    public function getName(): string
+    {
+        return 'ezpublish.rich_text';
```

Comment:
I've noticed it's called `richtext`, `ezrichtext` or `rich_text`. Since it's targeted at 3.0 release, can't we streamline it?

Reply:
> I've noticed it's called `richtext`, `ezrichtext` or `rich_text`. Since it's targeted at 3.0 release, can't we streamline it?

Good catch, I must have missed this when copying it from somewhere else.
Current prefix for Twig extensions is `ezrichtext` so I've changed this to `ezrichtext.configuration` via 7f25704a2e9319ea3dcedf0f7695d2fa1222d728. It's in the form of `"ezrichtext"` here to follow Symfony bundle extension name.

We can add `ezplatform` prefix as well, but the other extensions needs alignment in this case.